app/views/claims/_check_your_answers_section.html.erb

```diff
@@ -14,7 +14,7 @@
       </dd>
 
       <dd class="govuk-summary-list__actions">
-        <%= link_to 'Change', claim_path(slug), class: 'govuk-link' %>
+        <%= link_to('Change', claim_path(slug), class: 'govuk-link') unless slug.nil? %>
```

Comment:
If there's no slug, should we even show the last `<dd>` element?

Reply:
It keeps the layout consistent. I was on the fence, though.